### PR TITLE
Implement identifier for live session

### PR DESCRIPTION
**Pull request by yuna878**
Added isLive field to indicate whether a session is active/inactive

### CODE REVIEW

File: src/SessionManager.js [hunk 1]
```diff
@@ -94,7 +94,7 @@ class SessionManager {
           const isSession = x && x.session;
           return isSession ? x.session.code === sessionCode || x.session.id === id : false;
       });
-      return socket !== undefined;
+      return socket !== undefined ? socket.isLive : false;
```
Comment (meganle) [inline src/SessionManager.js]: Rather than adding an `isLive` field, are you able to just check if `socket.current.poll === -1`?

Comment (yuna878) [reply src/SessionManager.js]: The definition of a 'live session' that we came up was that session becomes live whenever a live poll starts, session becomes inactive when there isn't a live poll && no admin is present in socket. So if I just check for `socket.current.poll === -1` it wouldn't check the presence of admin in socket.

Comment (meganle) [reply src/SessionManager.js]: Ah gotcha makes sense

File: src/SessionSocket.js [hunk 2]
```diff
@@ -380,6 +387,8 @@ export default class SessionSocket {
       this.current.answers = {};
       this.current.upvotes = {};
 
+      if (!this.isLive) this.isLive = true;
```
Comment (meganle) [inline src/SessionSocket.js]: You could prob just do `this.isLive = true` for brevity since you know it will be true regardless

**Pull request merged**